src/controllers/user.js

```diff
@@ -136,3 +136,27 @@ export const updateProfile = async (req, res) => {
     return sendMessageResponse(res, 500, 'Unable to update new user')
   }
 }
+
+export const createNote = async (req, res) => {
+  const teacherId = req.user.id
+  const studentId = Number(req.params.id)
+  const { content, isEditted } = req.body
+  const newNoteData = {
+    teacherId,
+    studentId,
+    content,
+    isEditted
```

Comment:
spelling? `isEdited`

Reply:
its @Alex90Jennings :D italian spelling?
i am going to correct it now.